### DISCUSSION TITLE
Costumes: Made Toggleable

### DISCUSSION THREAD

**Discussion by xUkarai** [chromiecraft/chromiecraft#6284] [Suggestions]
### Costumes: Made Toggleable (or Duration Clarified in Catalogue Listing)

As it currently stands, costumes purchased from the catalogue have a cooldown between uses. (About 25% uptime for the Uncommon Harpy, for instance.) 

**However, costume duration and downtime is not stated in the donation catalogue.**
I assumed cosmetics would always be useable outside of PvP, as that was their only advertised limitation. Even in-game vanity effects (such as Noggenfogger or Halloween visuals) last longer, without real monetary cost. 

Gleefully, I picked a costume, just to find out it was only 1/4th as fun as it could be. This restriction felt strange. Then frustrating.

**Picture this.**
I walk into an ice cream store. Let's call it Blizzard. 
There is an image of an ice cream cone. 
It costs €5, but it states it cannot be eaten in PvP. 
That's fine, I don't really PvP, I want to eat my ice cream cone in peace. 
It tastes like a Harpy Costume. That's my favorite kind!
**Then a bell rings, Bobby Kotick shoves me, and I drop my ice cream cone.**
I am now stuck in a time loop of enjoying _a bit_ of ice cream, dropping the rest, and waiting for it to happen all over again.

**What's the harm in removing the downtime?**

It's already stated it cannot be used in pvp.

On a server my guild migrated from, donation costumes were toggleable without a cooldown. The buff icon could be removed and was done so automatically for mounts and flightpaths, and could be reapplied as needed, to avoid awkward circumstances where morphs would not function. **It was 100% harmless fun.**

**Anecdotally, people would buy costumes if the cooldown was removed!!**

**TL;DR**
**My suggestions:**
1. Make the costumes toggleable, or dramatically reduce the cooldown (and fulfill my dreams of being some sort of CREATURE). It's more fun, more marketable, and has no impact on gameplay.
2. Update the catalogue information to include costume restrictions for future donators. **Warning: You will drop your ice cream every 15 minutes.**

**Comment by TheDarkLordSunderland** [comment]
I just want to be a murloc priest. 

let me be murloc priest

P.S.
I wouldn't buy another "ice cream" again

**Comment by vrroena** [comment]
![cover1](https://github.com/chromiecraft/chromiecraft/assets/149058107/8440cf4d-fef7-4275-819f-e6583fe3394d)

Big updoot from me. I love donating to private server projects, especially ones as cozy and well done as Chromie Craft. (If I don't pay a monthly sub, the money still goes somewhere into development!)

As it currently stands, I don't really see an incentive to buying these morphs. Halloween wands unfortunately have more duration, infinitely increasing their fun compared to the donation shop morphs. This was disappointing to find as I purchased a dryad morph prior to learning it was only 5 minutes long.

On a RP-PVE server, I'd love to run around as a restoration druid dryad! But right now, it doesn't feel worth the hassle of babysitting the cool down of the effect for only 5 mins of uptime. I'd really love for these to be "on/off" unlimited effects!

That being said, I unfortunately won't be buying any additional morphs until there's more incentive to them, otherwise 5 minutes isn't a lot of fun. :(
